### ISSUE TITLE
Cast error

### ISSUE THREAD

**Issue by lauriejim** [Vnkitaev/mongoose-float#6]
Hello!

I have an issue during the cast of `null` or `undefined` or `empty` value.

In your [cast function](https://github.com/Vnkitaev/mongoose-float/blob/master/lib/index.js#L15), if the [value is not a number](https://github.com/Vnkitaev/mongoose-float/blob/master/lib/index.js#L16) you fire an error.

So in my case I will have an error stored in my document. And I would like to store `null`.

Why we can not use the same [cast system as mongoose](https://github.com/Automattic/mongoose/blob/master/lib/cast/number.js) for numbers?

```
  if (val == null) {
    return val;
  }
  if (val === '') {
    return null;
  }

  if (typeof val === 'string' || typeof val === 'boolean') {
    val = Number(val);
  }
```

**Comment by lauriejim** [comment]
Hello @Vnkitaev !

I ping you to know if you can give me some time to check this topic please.

**Comment by dodo360** [comment]
Hello @lauriejim , okay, i will fix it

**Comment by lauriejim** [comment]
Awesome thank you @Vnkitaev ! It will help the Strapi community.
Appreciate.

**Comment by lauriejim** [comment]
@Vnkitaev any new please.
It's really fast to fix and publish on npm since I give you all stuff to did it.

Really need this fix please.
Thank you for your time.

**Comment by dodo360** [comment]
@lauriejim fixed in 1.0.4, sorry for long waiting

**Comment by lauriejim** [comment]
I will check that on Strapi. Thank you!
